hoot-core/src/main/cpp/hoot/core/io/HootNetworkRequest.cpp

```diff
@@ -167,6 +167,9 @@ bool HootNetworkRequest::_networkRequest(QUrl url, const QMap<QNetworkRequest::K
   if (QNetworkReply::NoError != reply->error())
   {
     _error = reply->errorString();
+    //  Remove authentication information if present
+    if (request.url() != url)
+      _error.replace(request.url().toString(), url.toString(), Qt::CaseInsensitive);
```

Comment:
This removes the authentication information from the URL in the error string.  `request` has the full URL including `user:password@` while `url` didn't.

Reply:
Nice.